backend/app/services/wiki_service.py

```
@@ -0,0 +1,740 @@
+            task_user = main_db.query(User).filter(User.id == task_user_id).first()
+            if not task_user:
+                raise HTTPException(
+                    status_code=404,
+                    detail=f"Wiki task user (ID: {task_user_id}) not found. Please check WIKI_DEFAULT_USER_ID in your .env file",
+                )
+
+            try:
+                task_kinds_service.create_task_or_append(
+                    db=main_db, obj_in=task_create, user=task_user, task_id=task_id
+                )
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)